src/Interactive/Features/Interactive/Core/InteractiveHost.cs

```diff
@@ -353,13 +361,17 @@ private void ReportProcessExited(Process process)
                     }
                     else
                     {
+                        if (previousService == null) { return default; }
```

Comment:
>{ return default; } [](start = 53, length = 19)

style: braces on separate lines

Reply:
This needs to go after `newService.Dispose(joinThreads: false);`

---
In reply to: [188362651](https://github.com/dotnet/roslyn/pull/26770#discussion_r188362651) [](ancestors = 188362651)